style/style.css

```diff
@@ -0,0 +1,668 @@
+  .event-date {
+    margin-top: 64px;
+    padding-left: 138px;
+    padding-right: 6px;
+  }
+
+  .event-location {
+    font-weight: 700;
+    margin-top: 64px;
+    padding-left: 138px;
+    padding-right: 6px;
+  }
+}
+
+@media (max-width: 767px) {
+  .event-date {
+    margin-top: 30px;
+    padding-left: 30px;
+    padding-right: 1px;
+    width: 328px;
+    height: 36.76px;
+    font-size: 20px;
+    line-height: 0px;
+    text-align: center;
+    display: flex;
+    letter-spacing: -0.08em;
+  }
+
+  .event-location {
+    margin-top: 3px;
+    padding-left: 3px;
+    padding-right: 1px;
+    width: 100%;
+    font-size: 11px;
+    line-height: 0px;
+    text-align: center;
+    font-weight: 600;
+    display: flex;
+    letter-spacing: -0.02em;
+    padding-top: 1px;
+  }
+
+  .btn-mobile {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 900;
+    font-size: 16px;
+    line-height: 28px;
+    display: flex;
+    align-items: center;
+    letter-spacing: -0.01em;
+    color: #fff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    flex-direction: row;
+    justify-content: center;
+    margin: 20px 10px;
+    width: 300px;
+    height: 60px;
+    background: #ff4f38;
+  }
+}
+
+.custom-section {
+  height: 587px;
+  padding-top: 75px;
+  background-image: url(../images/diamond.png);
+  background-color: #282b32;
+  color: #fff;
+  position: relative;
+}
+
+.main-program-row {
+  margin-bottom: 40px;
+}
+
+.main-program-title {
+  position: relative;
+  display: inline-block;
+}
+
+.underline {
+  position: absolute;
+  bottom: -2px;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 40px;
+  height: 3px;
+  background-color: #a04038;
+}
+
+.see-program-link {
+  margin-top: 75px;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.01em;
+  text-decoration-line: underline;
+  color: #fff;
+}
+
+.see-program-link .underline {
+  display: inline-block;
+  width: 100%;
+  height: 2px;
+  background-color: #000;
+}
+
+.card {
+  background-color: rgba(255, 255, 255, 0.1) !important;
+  padding: 15px 9px;
+  align-items: center;
+  text-align: center;
+  width: 105%;
+  margin-top: 5px;
+}
+
+.card-title {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 28px;
+  align-items: center;
+  letter-spacing: -0.01em;
+  color: #ff5438;
+}
+
+.partner-title {
+  position: relative;
+  display: inline-block;
+}
+
+.partner-title .underline {
+  width: 51px;
+  height: 0;
+  margin-top: 55px;
+  border: 2px solid #a04038;
+}
+
+.partner-logo {
+  /* width: 200px;
+height: 75px; */
+  margin-top: 70px;
+}
+
+.partner {
+  display: block;
+  padding: 4% 10%;
+  background-color: #414246;
+  color: #fff;
+}
+
+/* Responsive adjustments for mobile */
+@media only screen and  (max-width: 600px) {
```

Comment:
- [ ] Hi, I noticed that your CSS styling is not built as mobile-first. You are using `@media (max-width:)` to adjust your layout for smaller screens, but this is not the best practice. Mobile-first design means that you start with the smallest screen size and then use `@media (min-width:)` to add more styles for larger screens. This will make your code more efficient and responsive, and also improve your performance and user experience. I suggest you refactor your CSS code to follow the mobile-first approach and see how it improves your design. You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Responsive/Mobile_first. Let me know if you have any questions! 😊